cartography/intel/github/repos.py

```diff
@@ -17,6 +17,9 @@
 
 logger = logging.getLogger(__name__)
 
+# (ryan-lane): To workaround a bug in the graphql API (see:
+# https://github.community/t/user-node-is-null-if-email-is-private-or-unset/179323)
+# email has been removed from the collaborators->nodes portion of this query
```

Comment:
Ah. I think you've based this incorrectly. Your change should be based on master, not on paypay-main.